bgp/local_tests/ygnmi_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package local_test
```

Comment:
i wonder if it's worth making an Ondatra binding for this behavior instead. add todo or task?

Reply:
Would an Ondatra binding also require a testbed and port specifications? For BGP tests there is no need for the dataplane, which of Ondatra's functionalities do you think we can make use of if we introduced the binding?

Reply:
I'll add a comment to consider doing this.